app/services/participants/sync_dqt_induction_start_date.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Participants
+  class SyncDqtInductionStartDate < BaseService
```

Comment:
`Sync` implies it will get the date from the Dqt, but this service obj receives the date when instantiated. Would `SetDqtInductionStartDate` read better?

Reply:
I agree the name could be more accurate but that was Fabio's choice. The issue now is that this PR is extracted from the big one and changes to this one would need to be replicated in the other as well.